app/grandchallenge/evaluation/models.py

```diff
@@ -475,6 +483,25 @@ class Phase(FieldChangeMixin, HangingProtocolMixin, UUIDModel):
         blank=True,
         help_text="The output interfaces that the algorithms for this phase must use",
     )
+    selectable_gpu_type_choices_evaluation: list[str] = models.JSONField(
+        blank=True,
+        null=True,
+        default=list,
+        help_text=(
+            "The GPU type choices that challenge admins will be able to set for the "
+            f"evaluation method. Options are {GPUTypeChoices.names}.".replace(
```

Comment:
I think this should be the values, not the names, as we want to match how these are represented in the database.

Reply:
The values should never be changed as that would involve a data migration. It's still there for historical reasons. Having it as an empty string is fine - only site admins will be modifying this. If we ever open it up to others we will anyway need to rework the widget as we do not want people editing JSON fields directly, has proven to be difficult for users.

Reply:
Ah of course!